tests/dummy/app/pods/expansion-type/template.hbs

```diff
@@ -0,0 +1,17 @@
+{{frost-list
+  class='demo'
+  hook='demo'
+  item=(component 'list-item')
+  itemExpansion=(component 'list-item-expansion')
+  items=items
+  expandedItems=expandedItems
+  selectedItems=selectedItems
+  onExpansionChange=(action 'onExpansionChange')
+  onSelectionChange=(action 'onSelectionChange')
+  sorting=(component 'frost-sort'
+    sortOrder=sortOrder
+    sortingProperties=sortingProperties
+    onChange=(action 'onSortingChange')
+  )
+  expansionType='initial'
```

Comment:
Please change this to be a property from the controller rather than a hard-coded string in the template.  How the value in the controller gets set is to import _app/utils/expansion-types.js_.  In this way we can demonstrate how can use the enum available as an export from this addon also as the input so that the entire usage is key-based and then if we ever want to change the value we can without breaking anybody's code, assuming they passed around the same references.